stubs/Parsers/ConfigurationFileParser.swift

```diff
@@ -15,29 +15,28 @@ class ConfigurationFile {
     }
 
     let indexSearchParameters: [String: Any]
-    let stubsModificationRules: [String: Any]
-    let indexModificationRules: [String: Any]
+    let stubsModificationRules: [String: Any]?
+    let indexModificationRules: [String: Any]?
 
     init(configurationFileURL: URL) {
         do {
             let rulesFileData = try Data(contentsOf: configurationFileURL)
             let object = try JSONSerialization.jsonObject(with: rulesFileData, options: [])
 
             guard let dictionary = object as? [String: Any],
-                let indexSearchParameters = dictionary[Constant.indexSearchParametersKey] as? [String: Any],
-                let stubsModificationRules = dictionary[Constant.stubsModificationRulesKey] as? [String: Any],
-
-            let indexModificationRules = dictionary[Constant.indexModificationRulesKey] as? [String: Any] else {
-                printError("Incorrect structure of: \n \(object)")
-                abort()
+                  let indexSearchParameters = dictionary[Constant.indexSearchParametersKey] as? [String: Any] else {
+                fatalError("index_search_parameters must exist within the configuration structure: \n \(object)")
             }
 
             self.indexSearchParameters = indexSearchParameters
-            self.stubsModificationRules = stubsModificationRules
-            self.indexModificationRules = indexModificationRules
+            self.stubsModificationRules = dictionary[Constant.stubsModificationRulesKey] as? [String: Any]
```

Comment:
There are times when we don't care to update indices. Other times we don't care to update stubs. This makes the script a bit more flexible.

Reply:
Agree